alpaca/broker/client.py

```diff
@@ -36,9 +41,40 @@ def validate_account_id_param(account_id: Union[UUID, str]) -> UUID:
     return account_id
 
 
+class PaginationType(Enum):
+    """
+    An enum for choosing what type of pagination of results you'd like for BrokerClient functions that support
+    pagination.
+
+    * NONE: Requests that we perform no pagination of results and just return the single response the API gave us.
```

Comment:
I've been using this style for listing enum attributes.

data/enums.py
```
class Exchange(Enum):
  """The exchanges that provide data feeds to Alpaca
      Attributes:
          FTXU (str): FTX exchange
          CBSE (str): Coinbase exchange
          GNSS (str): Genesis exchange
          ERSX (str): ErisX exchange
          IEX (str): Investors Exchange
```

There's not much information on how to write docstrings for enums.

I'm not sure if we want to use the same style here. 🤔

Reply:
Ah yeah I guess Exchanges renders in the docs like:
![echange-docs](https://user-images.githubusercontent.com/2347730/165584033-0db80be4-e6ab-44e7-a226-a6eb12b3d3bf.png)

Where this renders as 
![pagination](https://user-images.githubusercontent.com/2347730/165584083-8bb135d4-3865-43fc-b2eb-a05b1c4883a3.png)

Reply:
I'm good making it like the above but the repeated `type str`  feels a little annoying. Esp since PaginationType extends str and Enum already so its clear that its values must be str :thinking: